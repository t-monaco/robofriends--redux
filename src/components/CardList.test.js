import React from 'react';
import { shallow } from 'enzyme';
import CardList from './CardList';

it('renders without crashing', () => {
    const robots = [
        {
            id: 1,
            name: 'Tom Rider',
            email: 'tomrider@mail.com'
        }
    ]
    expect(shallow(<CardList robots={robots}/>)).toMatchSnapshot();
});
