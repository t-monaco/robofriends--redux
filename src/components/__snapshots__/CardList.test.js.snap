// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="flex justify-center flex-wrap"
>
  <Card
    email="tomrider@mail.com"
    id={1}
    key="0"
    name="Tom Rider"
  />
</div>
`;
